prql-compiler/prqlc/src/cli.rs

```diff
@@ -292,7 +292,7 @@ impl Command {
         // it's confusing whether it's waiting for input or not. This
         // offers the prompt.
         if input.is_stdin() && atty::is(atty::Stream::Stdin) {
-            println!("Enter PRQL, then ctrl-d:\n");
+            println!("# Enter PRQL, then press ctrl-d or ctrl-z (windows) to compile:\n");
```

Comment:
```suggestion
            eprintln!("# Enter PRQL, then press ctrl-d or ctrl-z (windows) to compile:\n");
```

...I think this will solve the issue above

Reply:
FYI we could use this approach to print the correct string: https://github.com/PRQL/prql/blob/e32c5864e3ca04987849e2397d956a0c7a599447/prql-compiler/prqlc/src/cli.rs#L508